### ISSUE TITLE
Unable to delete table with a keyboard only when editor contains only table

### ISSUE THREAD

**Issue by natanielcz** [JiHong88/SunEditor#254]
Affected browsers: Mozilla Firefox
1. Steps to reproduce: Create a table without line before and after (or just insert html from below)
```html
<table style="width: 100%;">
    <tbody>
        <tr>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
        </tr>
        <tr>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
        </tr>
        <tr>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
            <td>
                <br>
            </td>
        </tr>
    </tbody>
</table>
```

2. Select table with mouse or Ctrl+A
3. Press delete to remove the table

On gif below on the left side is Google Chrome and on the right Mozilla Firefox

![UkW9ghvOxB](https://user-images.githubusercontent.com/1231703/76857718-4c9c2880-6856-11ea-9fa0-f80ed20c58bb.gif)
